Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you remove all non-related files from your root directory to make it more professional for example: `1.0` file,...